test/specs/util/validate.js

```
@@ -10,7 +10,7 @@ define(['util/validate'], function(validate) {
 
     Generic: {
       good: [' ', 'anything', '!@#$%^&*()-=', '\''],
-      bad: ['<script type="jarvascript">document.write', '', null, undefined]
+      bad: ['<script type="jarvascript">document.write']
```

Why were these values taken out? `Generic` should fail given `null` or `undefined`.